numba/tests/test_unicode.py

```diff
@@ -2401,5 +2403,58 @@ def test_unicode_literal_stopiteration_iter(self):
                 f()
 
 
+class TestUnicodeAuxillary(BaseTest):
+
+    def test_ord(self):
+        pyfunc = ord_usecase
+        cfunc = njit(pyfunc)
+        for ex in UNICODE_EXAMPLES:
+            for a in ex:
+                self.assertPreciseEqual(pyfunc(a), cfunc(a))
+
+    def test_ord_invalid(self):
+        self.disable_leak_check()
+
+        pyfunc = ord_usecase
+        cfunc = njit(pyfunc)
+
+        # wrong number of chars
+        for func in (pyfunc, cfunc):
+            for ch in ('', 'abc'):
+                with self.assertRaises(TypeError) as raises:
+                    func(ch)
+                self.assertIn('ord() expected a character',
+                              str(raises.exception))
+
+        # wrong type
+        with self.assertRaises(TypingError) as raises:
+            cfunc(1.23)
+        self.assertIn('Invalid use of Function', str(raises.exception))
+
+    def test_chr(self):
+        pyfunc = chr_usecase
+        cfunc = njit(pyfunc)
+        for ex in UNICODE_EXAMPLES:
+            for x in ex:
+                a = ord(x)
```

Comment:
It might be worth adding explicit tests for  the upper and lower boundaries, i.e. `0` and `0x10ffff`.

Reply:
they are covered somewhat by the asserting of invalid values, but its 2 lines so no problem